screens/dashboard/index.js

```diff
@@ -227,7 +227,7 @@ class Dashboard extends Component<Props> {
                             </View>
                             <View style={styles.smallerBlock}>
                                 <Text style={styles.bigBlock}/>
-                                <Text style={[styles.money, styles.start]}>{'$500'}</Text>
+                                <Text style={[styles.money, styles.start]}>{'$' + incentivesAvailable}</Text>
```

Comment:
This works just fine.

It's also common to use string interpolation too: `$${incentivesAvailable}`